python/python/bystro/cli/cli.py

```diff
@@ -177,6 +177,7 @@ def stream_file_cli(args: argparse.Namespace) -> None:
         output=args.output,
         key_path=args.key_path,
         out_dir=args.out_dir,
+        write_stdout=not args.out_dir
```

Comment:
If the user doesn't want to write to an output directory, the default should be stdout, not outputting a generator